common/src/bitset.rs

```diff
@@ -369,7 +366,11 @@ impl ReadSerializedBitSet {
         b & (1u8 << shift) != 0
     }
 
-    /// Returns the max_value.
+    /// Maximum value the bitset may contain.
+    /// (Note this is not the maximum value contained in the set.)
+    ///
+    /// A bitset has an intrinsic capacity.
+    /// It only stores elements within [0..max_value).
```

Comment:
this comment suggests to maybe rename it to capacity

Reply:
Yeah it does feel weird right? Capacity is not great either though.

Reply:
yes I agree

Reply:
maybe limit?

Reply:
`num_bits`, `num_capacity_bits` ?

Reply:
`width`!

Reply:
`bit_width` is quite good actually.